doc/manual/src/language/constructs.md

```diff
@@ -158,6 +158,8 @@ in
 
 both evaluate to `{ names = [ "a" "b" ]; }`.
 
+Inherited variables shadow existing variables from outer scopes.
```

Comment:
As mentioned by @Ericson2314  (https://github.com/NixOS/nix/pull/10620#issuecomment-2085427271), `inherit`ed attributes works like a syntactic sugar for `name = name`.  I think it is misleading to add words here. (Because `inherit` makes nothing different from normal attributes).

Reply:
Done, removed.